system/bind/templates/deployment.yaml

```diff
@@ -86,6 +86,11 @@ spec:
         {{ if .Values.extra_env }}
 {{ toYaml .Values.extra_env | indent 8 }}
         {{ end }}
```

Comment:
and this `end` move down under `envFrom` section.

Reply:
```envFrom``` is unconditional. If there are no secret env vars then it will just stay empty.